src/graphql/public/schema.graphql

```diff
@@ -1011,6 +1014,7 @@ type Query {
   businessMapMarkers: [MapMarker]
   currencyList: [Currency!]!
   globals: Globals
+  isFlashNpub(npub: String): Boolean
```

Comment:
1. I do not think we should be leaving the list of Flash npubs unauthed. the less info we leak about our users the better.

2. Nostr user details should always come from relays or the multiplexer. I can't think of a reason why we would need to store any on our backend that already exists on nostr.

Reply:
1. If we want to add authorization, we should think about how to implement it. Graphql authorization uses a token obtained via Kratos. Some options:
i. We could potentially create a service account in kratos for nostr which gives us access to this endpoint, but this may cause headaches because the Graphql api authorization is designed for flash-mobile users. 
ii. We could create a separate api endpoint (outside the gql schema), which would not require authorization if we do not expose it publically (only cluster). 

2. :heavy_check_mark:

Reply:
@islandbitcoin list of flash npubs is going to be un-authed anyways? cause they'll be on the .well-known url? does it matter if the API leaks this data as well? I don't think this is a huge concern. I do want to add "username" on this API as well. so that I can power the well-known API from this endpoint directly.

I don't think the "leak" concern is valid in this case

Reply:
good point, they will be on .well-known url. and username will be there too. 

unauthed query could work, as long as you can't think of any way for someone to socially engineer that data.